opm/output/data/CellDataContainer.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+  Copyright 2016 Statoil ASA.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <algorithm>
+
+#include <opm/output/data/CellDataContainer.hpp>
+#include <opm/output/data/Cells.hpp>
+
+
+namespace Opm {
+
+    CellDataContainer::CellDataContainer( std::vector<data::CellData> data ) {
+        for (const auto& elm : data)
+            this->data.push_back( std::move(elm) );
```

Comment:
The code has been changed to use `std::initializer_list<>` - that was actually the intention all the way; I just did not understand how to it first. I will add documentation.